src/plugins/data/common/search/aggs/buckets/histogram.ts

```diff
@@ -159,8 +170,9 @@ export const getHistogramBucketAgg = ({
       },
       {
         name: 'maxBars',
+        default: getConfig(UI_SETTINGS.HISTOGRAM_MAX_BARS),
```

Comment:
it will cause a regression for integers values

![image](https://user-images.githubusercontent.com/20072247/91351704-67a19a00-e7f1-11ea-9c79-8690c7a5e5e1.png)

Reply:
6 should be the correct interval for that case by default.